TWTValidation/JSON Validator/Parser/TWTJSONRemoteSchemaManager.m

```diff
@@ -0,0 +1,95 @@
+//
+//  TWTJSONRemoteSchemaManager.m
+//  TWTValidation
+//
+//  Created by Jill Cohen on 10/19/15.
+//  Copyright © 2015 Ticketmaster Entertainment, Inc. All rights reserved.
+//
+
+#import "TWTJSONRemoteSchemaManager.h"
+
+#import <TWTValidation/TWTJSONSchemaASTCommon.h>
+#import <TWTValidation/TWTJSONSchemaParser.h>
+
+
+@interface TWTJSONRemoteSchemaManager ()
+
+@property (nonatomic, strong) NSMutableDictionary *filePathsToJSONSchemaTopLevelNodes;
+
+@end
+
+
+@implementation TWTJSONRemoteSchemaManager
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        _filePathsToJSONSchemaTopLevelNodes = [[NSMutableDictionary alloc] init];
+    }
+    return self;
+}
+
+
+- (BOOL)attemptToConfigureFilePath:(NSString *)filePath onReferenceNode:(TWTJSONSchemaReferenceASTNode *)referenceNode
+{
+    if (self.filePathsToJSONSchemaTopLevelNodes[filePath]) {
+        return YES;
+    }
+
+    if ([filePath isEqualToString:TWTJSONSchemaKeywordDraft4Path]) {
+        filePath = [[NSBundle bundleForClass:[self class]] pathForResource:@"JSONSchemaDraft4" ofType:@"json"];
```

Comment:
Wanna make this filename a constant at the top of the file? That way, we can find it more easily if we need to rename it.